REFERENCE.md

```diff
@@ -546,26 +551,55 @@ Default value: `undef`
 Locks package from updates.
 
 * **Note** The resource title must use the format
+By default on CentOS 7 the following format is used.
 "%{EPOCH}:%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}".  This can be retrieved via
 the command `rpm -q --qf '%{EPOCH}:%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}'.
 If "%{EPOCH}" returns as '(none)', it should be set to '0'.  Wildcards may
 be used within token slots, but must not cover seperators, e.g.,
 '0:b*sh-4.1.2-9.*' covers Bash version 4.1.2, revision 9 on all
 architectures.
+By default on CentOS 8 and newer the resource title  to just set the
+package name.
+If future is set to true on CentOS 7 then it behaves like CentOS 8
 
 * **See also**
 http://man7.org/linux/man-pages/man1/yum-versionlock.1.html
 
 #### Examples
 
-##### Sample usage
+##### Sample usage on CentOS 7
 
 ```puppet
 yum::versionlock { '0:bash-4.1.2-9.el6_2.*':
   ensure => present,
 }
 ```
 
+##### Sample usage on CentOS 8
+
+```puppet
+yum::versionlock { 'bash':
+  ensure => present,
+  version => '4.1.2',
+  release => '9.el6',
```

Comment:
one probably wouldn't lock el6 version on CentOS 8?

Reply:
This and other el6, 7 , 8 examples made consistent with example package names to contain el6, 7 or 8.